docs/smart-contracts.md

```diff
@@ -0,0 +1,157 @@
+Dragonchain Smart Contracts
+===
+
+Smart Contracts in Dragonchain allow custom business rules to be privately deployed to your node's blockchain, 
+similar to stored procedures in a traditional database. Unlike stored procedures however, Dragonchain Smart Contracts 
+are signed, provide higher level features for integrating with blockchain data and are datastore agnostic. 
+
+Since Dragonchain is a hybrid permissioned / public ledger, it's assumed that smart contracts are written and signed by 
+trusted parties. This loosens requirements that would apply to a public blockchain since your smart contracts only run 
+on trusted nodes. As such, Dragonchain doesn't require a proprietary VM and associated languages for executing smart 
+contracts avoiding the notion of gas and cost per instruction altogether. This is suitable for business-critical 
+applications since operational costs are fixed and independent of currency speculation while still providing a 
+cryptographically secured immutable database environment.
+
+Polyglot smart contracts are on the roadmap, the current implementation is Python based.
+
+Smart Contract (SC) Types
+===
+
+- Transaction SC (TSC) - Primary business logic, runs on applicable transactions during phase 1
+- Subscription SC (SSC) - Processes blocks received from peers via subscriptions
+- Library SC (LSC) - Dragonchain core provided smart contracts
+- Broadcast Receipt SC (BSC) - Reacts to verification data received from higher phase nodes
+
+Transaction SC
+===
+
+The TSC is meant for implementing custom business logic on-top of transactions your node receives. It provides the final verification and guarantee your business domain transactions are accurate and valid before being accepted into the current block and broadcast to the network. 
```

Comment:
Change to (changes in bold):

The TSC is meant for implementing custom business logic **to process** transactions your node receives. It provides the final verification and guarantee **that** your business domain transactions are accurate and valid **according to your own business rules** before being accepted into the current block and broadcast to the network.